src/node/markdown/plugins/lineNumbers.ts

```diff
@@ -3,10 +3,22 @@
 
 import MarkdownIt from 'markdown-it'
 
-export const lineNumberPlugin = (md: MarkdownIt) => {
+export const lineNumberPlugin = (
+  md: MarkdownIt,
+  isLineNumberOption: Boolean
+) => {
   const fence = md.renderer.rules.fence!
   md.renderer.rules.fence = (...args) => {
     const rawCode = fence(...args)
+    const [tokens, idx] = args
+    const token = tokens[idx]
+    const attr = token.attrs && token.attrs[0]
```

Comment:
This logic won't work if markdown-it-attrs is disabled. As a fallback, you need to manually check token.info too for that option (refer highlightLines plugin for example, we had done similar stuff there). Also, have you checked if this works with line highlighting and snippets import? If yes, then consider adding some example to the docs.

Reply:
@brc-dd On disabling `markdown-it-attrs`, I noticed that on the top right corner of code block where we are displaying the language for the code, the `{1,4,6-8}` - line highlightning syntax is also coming up. Is this intended or a bug ?
![Screenshot 2022-09-24 132412](https://user-images.githubusercontent.com/61521805/192111043-9ceccf26-aa37-458a-b7bd-755a83515a7e.png)

Reply:
Yeah it's a bug.

Reply:
I have resolved the above bug and also line numbers are now working even if `markdown-it-attrs` is disabled. Please, let me know if you see any issues :)